video/out/vo_gpu_next.c

```diff
@@ -265,7 +265,7 @@ static void update_overlays(struct vo *vo, struct mp_osd_res res, double pts,
         ok = pl_tex_upload(p->gpu, &(struct pl_tex_transfer_params) {
             .tex        = entry->tex,
             .rc         = { .x1 = item->packed_w, .y1 = item->packed_h, },
-            .stride_w   = item->packed->stride[0] / tex_fmt->texel_size,
+            .row_pitch   = item->packed->stride[0] / tex_fmt->texel_size,
```

Comment:
This is wrong, `row_pitch` should be in bytes - remove the division by the texel size

(Note: wrong whitespace)